src/gt4py/frontend/gtscript_frontend.py

```diff
@@ -880,6 +880,12 @@ def visit_Assign(self, node: ast.Assign) -> list:
 
         return result
 
+    def visit_AugAssign(self, node: ast.AugAssign):
+        """Implement left <op>= right in terms of left = left <op> right."""
+        binary_operation = ast.BinOp(left=node.target, op=node.op, right=node.value)
```

Comment:
Location of the AugAssign node should be passed to the newly created nodes

Reply:
Interestingly there is no test that ensures all the nodes have a location.

Reply:
That would actually be a useful test for the Definition IR. I would make that a different PR though.